generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -5,12 +5,16 @@
     kill_vm_on_error = yes
     login_timeout = 240
     reboot_count = 25
```

Comment:
Could we reduce the value of reboot_count ?   It is time-consuming.  IMHO, 10 times is enough.
And, it is not related to the new case in fact, just a suggestion, it is up to you.